### PR TITLE
refactor: Suffix issue in responsive slider & design CSS changes in customizer

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Suffix comes on next line in case
- Radio image control's SVG design slight updated
- Unit suffix design in slider & responsive slider controls
- Responsive Icon for Background control is aligned next to label as per design changes

### Screenshots
- https://a.cl.ly/llu9yQYG
- https://a.cl.ly/RBu4d6Ee
- https://a.cl.ly/nOulNwA5
- https://a.cl.ly/OAugrx21

### Types of changes
- Non-breaking changes
- Related to customizer-redesign

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by premanshup** [comment]
@imnavanath Please pull latest code and resolve merge conflicts.

File: inc/assets/css/ast-builder-customizer-rtl.css [hunk 1]
```diff
@@ -1362,14 +1362,15 @@ li#customize-control-astra-settings-header-desktop-items, li#customize-control-a
...
+    width: 80px;
+    height: 54px;
 }
 
 #customize-control-astra-settings-header-trigger-icon .components-button.is-tertiary.active-radio svg, #customize-control-astra-settings-header-trigger-icon .components-button.is-tertiary svg,
 #customize-control-astra-settings-header-trigger-icon svg {
     width: 52px;
-    height: 50px;
+	height: 50px;
+	margin-left: 10px;
```
Comment (premanshup) [inline inc/assets/css/ast-builder-customizer-rtl.css:1373]: @imnavanath Please check the alignemnt.

File: inc/customizer/custom-controls/assets/css/unminified/custom-controls.css [hunk 2]
```diff
@@ -1446,9 +1420,65 @@ input[type="text"].select2-search__field {
...
+  border-width: 1px 1px 1px 0;
+  border-style: solid;
+  border-color: #ddd;
+  border-radius: 0 2px 2px 0;
+}
+.customize-control-ast-slider .components-input-control__container div.components-input-control__backdrop, .customize-control-ast-responsive-slider .components-input-control__container div.components-input-control__backdrop {
+ border: none;
+ border-radius: 2px 0 0 2px;
+ box-shadow: none;
 }
```
Comment (premanshup) [inline inc/customizer/custom-controls/assets/css/unminified/custom-controls.css:1566]: check the alignment.

**Comment by imnavanath** [comment]
> @imnavanath Please pull latest code and resolve merge conflicts.

Done @premanshup, branch is upto date now.

File: inc/customizer/custom-controls/assets/css/unminified/custom-controls.css (hunk 2 above)
Comment (imnavanath) [reply inc/customizer/custom-controls/assets/css/unminified/custom-controls.css]: Done @premanshup Thanks

File: inc/assets/css/ast-builder-customizer-rtl.css (hunk 1 above)
Comment (imnavanath) [reply inc/assets/css/ast-builder-customizer-rtl.css:1373]: Aligned correctly - https://a.cl.ly/BluYBBNB

**Review by premanshup — APPROVED** [review]

**Pull request merged**